install.sh

```diff
@@ -155,6 +155,8 @@ else
     FILE=~/.${shell}rc
 fi
 
+echo >> $FILE
```

Comment:
Adding a brief comment here explaining the rationale could be useful for our future selves :smile:

Reply:
Comment is added now :)